### ISSUE TITLE
适配请求 希望适配RISC-V架构的板卡设备

### ISSUE THREAD

**Issue by DSOE1024** [IceWhaleTech/CasaOS#1669]
希望适配RISC-V架构，现在有很多RISC-V板卡设备可以做NAS

**Comment by ETWang1991** [comment]
Got it

**Comment by LordCasser** [comment]
I agree with that. I'm trying to use VisionFive2 for a NAS. Everything seems OK, after I installed docker(apt install docker.io) and rclone (apt install rclone). Maybe they are not newest, but I think they are useful.
After all ,something got wrong.
![image](https://github.com/IceWhaleTech/CasaOS/assets/35390821/7925bfdb-e797-4061-b088-0efb1cfb6e38)
I think that because the install script install an amd64 version into my computer!
![image](https://github.com/IceWhaleTech/CasaOS/assets/35390821/c3912226-062c-4e04-9f6b-34fed202c27a)
I will try to rebuild them if I have some time.
But I hope these can help you to fix it.
